build.yaml

```diff
@@ -6,14 +6,25 @@ builders:
     auto_apply: all_packages
     build_to: cache
 
+  # Builder just for this package.
+  overReactLocalBuilder:
+    import: "package:over_react/builder.dart"
+    builder_factories: ["overReactBuilder"]
+    build_extensions: {".dart" : [".over_react.g.dart"]}
+    auto_apply: none
+    build_to: source
+
 targets:
   $default:
     builders:
-      over_react|overReactBuilder:
+      over_react|overReactLocalBuilder:
+        enabled: true
         generate_for:
           exclude:
             # This tests un-built behavior, and therefore should not be built
             - "test/over_react/component_declaration/builder_helpers_test.dart"
+      over_react|overReactBuilder:
+        enabled: false
```

Comment:
So this only disables it for this package? Is that how `targets.$default` works?

Reply:
Yes, that only does this for this package. The `$default` target specifies the behavior that consumers will use when running builds on this package: https://github.com/dart-lang/build/tree/master/build_config#dividing-a-package-into-build-targets

Reply:
Consuming packages' builds are unaffected by that (only applies to over_react)